lib/lrama/warnings/redefined_rules.rb

```diff
@@ -14,8 +14,8 @@ def initialize(logger, warnings)
       def warn(grammar)
         return unless @warnings
 
-        grammar.parameterizing_rule_resolver.redefined_rules.each do |rule|
-          @logger.warn("parameterizing rule redefined: #{rule}")
+        grammar.parameterized_resolver.redefined_rules.each do |rule|
+          @logger.warn("parameterized rule redefined: #{rule}")
```

Comment:
MEMO) I had a little difficulty here. Since this is a warning against “parameterized rules” rather than function names, I did not use Parameterizing. I don't have any strong opinions on this, so I will revert back if it feels different.